src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,73 @@
+import { User } from '../../types/User';
+import { actions } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const closeModal = () => dispatch(actions.removeTodo());
+
+  useEffect(() => {
+    if (currentTodo) {
+      getUser(currentTodo.userId).then(user => {
+        setSelectedUser(user);
+      });
+    }
+  }, []);
+
   return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
-
-      <Loader />
-
-      <div className="modal-card">
-        <header className="modal-card-head">
-          <div
-            className="modal-card-title has-text-weight-medium"
-            data-cy="modal-header"
-          >
-            Todo #3
-          </div>
+    currentTodo && (
+      <div className="modal is-active" data-cy="modal">
+        <div className="modal-background" />
 
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <button type="button" className="delete" data-cy="modal-close" />
-        </header>
+        {!selectedUser ? (
+          <Loader />
+        ) : (
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <div
+                className="modal-card-title has-text-weight-medium"
+                data-cy="modal-header"
+              >
+                Todo #
+                {currentTodo.id}
+              </div>
 
-        <div className="modal-card-body">
-          <p className="block" data-cy="modal-title">fugiat veniam minus</p>
+              <button
+                aria-label="close"
+                type="button"
+                className="delete"
+                data-cy="modal-close"
+                onClick={() => closeModal()}
+              />
+            </header>
 
-          <p className="block" data-cy="modal-user">
-            {/* For not completed */}
-            <strong className="has-text-danger">Planned</strong>
+            <div className="modal-card-body">
+              <p className="block" data-cy="modal-title">
+                {currentTodo.title}
+              </p>
 
-            {/* For completed */}
-            <strong className="has-text-success">Done</strong>
-            {' by '}
-            <a href="mailto:Sincere@april.biz">Leanne Graham</a>
-          </p>
-        </div>
+              <p className="block" data-cy="modal-user">
+                {currentTodo.completed ? (
+                  <strong className="has-text-success">Done</strong>
+                ) : (
+                  <strong className="has-text-danger">Planned</strong>
+                )}
+
+                {' by '}
+
+                <a href={`mailto: ${currentTodo && selectedUser.email}`}>
+                  {currentTodo && selectedUser.name}
+                </a>
```

Comment:
why do we need these checks 
```suggestion
                <a href={`mailto: ${selectedUser.email}`}>
                  {selectedUser.name}
                </a>
```
if we add render this information only when currentTodo exists
<img width="341" alt="image" src="https://github.com/mate-academy/react_redux-list-of-todos/assets/70220667/8d4b5098-e4e3-41ba-b59d-12f0dcd8b452">